source/_docs/migrate-drupal-gardens.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Migrating a Drupal Gardens Site to Pantheon
+description: Get all the details you need to know to successfully migrate your site away from Drupal Gardens.
+categories: [drupal]
+tags: [migrate]
+keywords: drupal, pantheon, drupal gardens, import
+---
+Drupal Gardens is ending support on August 1, 2016. This article walks you through exporting and migrating your site from Drupal Gardens to Pantheon.
+
+## Export a Site from Drupal Gardens
+
+1. Log in to your [Drupal Gardens account](https://www.drupalgardens.com/user/login).
+2. Go to **My Sites**.
```

Comment:
@rachelwhitton In that doc midway through the page, they say: "You can also export your website from the My sites page by clicking Export site in your website's Actions menu list."